com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Bob.java

```diff
@@ -467,7 +486,7 @@ private static void addOption(Options options, String shortOpt, String longOpt,
         }
     }
 
-    private static Options getCommandLineOptions() {
+    public static Options getCommandLineOptions() {
```

Comment:
made public so that the editor will be able to use command line options to define editor script API to bob with docs and validation.